corehq/apps/data_dictionary/views.py

```diff
@@ -68,19 +80,26 @@ def generate_data_dictionary(request, domain):
 @toggles.DATA_DICTIONARY.required_decorator()
 def data_dictionary_json(request, domain, case_type_name=None):
     props = []
+    fhir_resource_type_name_by_case_type = {}
+    fhir_resource_prop_by_case_prop = {}
     queryset = CaseType.objects.filter(domain=domain).prefetch_related(
         Prefetch('properties', queryset=CaseProperty.objects.order_by('name'))
     )
+    if toggles.FHIR_INTEGRATION.enabled(domain):
+        fhir_resource_type_name_by_case_type, fhir_resource_prop_by_case_prop = load_fhir_resource_mappings(
+            domain)
     if case_type_name:
         queryset = queryset.filter(name=case_type_name)
     for case_type in queryset:
         p = {
             "name": case_type.name,
+            "fhir_resource_type": fhir_resource_type_name_by_case_type.get(case_type),
```

Comment:
note to self: even though the empty key-value pair wont hurt in the json response when toggle is not enabled, consider keeping it clean and adding only when toggle enabled